src/units.jl

```diff
@@ -110,7 +113,10 @@ end
 @add_prefixes F (f, p, n, μ, u, m)
 @add_prefixes Ω (n, μ, u, m, k, M, G)
 @add_prefixes ohm (n, μ, u, m, k, M, G)
+@add_prefixes S (n, μ, u, m, k, M, G)
```

Comment:
What I’ve done so far (to prevent bloat) is only put in the units that people are actually using. What prefixes do you actually use in code? Is it just `S` or do you have examples of the use of prefixes?

Reply:
I'm keeping it on par with ohm as S = 1/ohm

Reply:
Oh I see. That makes sense. Maybe for the other ones though, reduce the number of prefixes? For all of the units my strategy has basically been to wait until someone wants a given prefix and then add it.

Reply:
I've dropped all the prefixes of henry.
For webers, m, n, and u are commonly used; I've dropped the rest.

Reply:
Cool, thanks!